resource/words/food.txt

```diff
@@ -40,4 +41,4 @@ tomato
 waffle
 walnut
 water
-wine
\ No newline at end of file
+wine
```

Comment:
A fantastic addition!

One issue though. It looks like you added an extra line at the end of the file. Because of the way that mangobyte parses this file, that means it will count this empty line as a word, and when you get a "food" item, sometimes you might just get "" (an empty string) in response. You can fix in either of these 2 ways:

1. Just remove this extra line at the end of the file.
2. Fix the function that parses this file so that it removes these empty strings. This would just be adding a line or two at line 28 of general.py